src/authentication/authentication.service.spec.ts

```diff
@@ -4,8 +4,8 @@ import { TokenService } from '../token/token.service';
 import { AuthenticationService } from './authentication.service';
 import { UserDTO } from '../dtos/user.dto';
 import { UserSessionDTO } from '../dtos/user-session.dto';
-import logWrapper from '../Logger/Logger.service';
-import { LogModule } from '../Logger/Logger.module';
+import { LogModule } from '@us-epa-camd/easey-common/logger';
+import { Logger } from '@us-epa-camd/easey-common/logger';
```

Comment:
FYI...these can be combined into a single import...
import { LogModule, Logger } from '@us-epa-camd/easey-common/logger';